lightning/src/chain/onchaintx.rs

```diff
@@ -347,7 +333,7 @@ impl<ChannelSigner: Sign> OnchainTxHandler<ChannelSigner> {
 
 		// Compute new height timer to decide when we need to regenerate a new bumped version of the claim tx (if we
 		// didn't receive confirmation of it before, or not enough reorg-safe depth on top of it).
-		let new_timer = Some(Self::get_height_timer(height, cached_request.timelock()));
+		let new_timer = Some(onchain_utils::get_height_timer(height, cached_request.timelock()));
```

Comment:
Why move this to `onchain_utils` if it is only used here? I think we should avoid "utils" modules as they tend to collect unrelated code.

Reply:
Right, made it part of the package interface as it's belong to it.